lib/rubocop/cop/rspec/described_class.rb

```diff
@@ -139,7 +139,7 @@ def offensive?(node)
           if style == :described_class
             offensive_described_class?(node)
           else
-            node.send_type? && node.method_name == :described_class
+            node.send_type? && node.method?(:described_class)
```

Comment:
Will this work with the earliest RuboCop version that we support, 0.68.1?

Reply:
With 0.68.1 installed `./spec/rubocop/cop/rspec/implicit_subject_spec.rb` fails several examples with:
```
      NoMethodError:
        undefined method `method?' for #<RuboCop::AST::BlockNode:0x00007ff05a4aaef0>
        Did you mean?  method
                       methods
```

Do you mind to keep those intact and disabling the cop that suggested this to `.rubocop_todo.yml`?

Reply:
Hmm, what more can if `bundle exec rubocop` passes?